CONTRIBUTING.md

```diff
@@ -0,0 +1,412 @@
+
+- **Community owned and maintained samples**: Code samples that have been
+  contributed by Knative community members. These samples might not receive
+  regular maintenance. It is possible that a sample is no longer current and is
+  not actively maintained by its original author. While we encourage a
+  contributor to maintain their content, we acknowledge that it's not always
+  possible for certain reasons, for example other commitments and time
+  constraints.
+
+While a sample might be out of date, it could still provide assistance and help
+you get up-and-running with certain use-cases. If you find that something is not
+right or contains outdated info, open an
+[Issue](https://github.com/knative/docs/issues/new). The sample might be fixed
+if bandwidth or available resource exists, or the sample might be taken down and
+archived into the last release branch where it worked.
+
+#### Choosing the correct branch
+
+It is likely that your docs contribution is either for new or changed features
+in the product, or for a fix or update existing content.
+
+- **New or changed features**: If you are adding or updating documentation for a
+  new or changed feature, you likely want to open your PR against the `master`
+  branch. All pre-release content for active Knative development belongs in
+  [`master`](https://github.com/knative/docs/tree/master/).
+
+- **Fixes and updates**: If you find an issue in a past release, for example a
+  typo or out-of-date content, you likely need to open multiple and subsequent
+  PRs. If not a followup PR, at least add the "`cherrypick` labels" to your
+  original PR to indicate in which of the past release that your change affects.
+
+  For example, if you find a typo in a page of the `v0.5` release, then that
+  page in the `master` branch likely also has that typo.
+
+  To fix the typo:
+
+  1.  Open a PR against the
+      [`master`](https://github.com/knative/docs/tree/master/) branch.
+  1.  Add one or more `cherrypick-#.#` labels to that PR to indicate which of
+      the past release branches should also be fixed. Generally, we only
+      maintain the most recent numbered release.
+  1.  If you want to complete the fix yourself (**best practice**), you then open a
+      subsequent PR by running `git cherry-pick [COMMIT#]` against the
+      `release-0.5`.
+      Where  `[COMMIT#]` is the commit of the PR that you merged in `master`.
+
+      Note: Depending on workload and available bandwidth, one of the Knative
+      team members might be able to help handle the `git cherry-pick` in order
+      to push the fix into the affected release branch(es).
+
+See a list of the available documentation versions in the
+[branches page](https://github.com/knative/docs/branches)
+of the `knative/docs` repo.
+
+## Assigning owners and reviewers
+
+For both documentation and code samples, you should assign your PR to a single
```

Comment:
likewise, IDK if most users have permissions to assign PRs, etc?

Reply:
This should be possible with `/assign`